src/Compilers/VisualBasic/Portable/Semantics/Operators.vb

```diff
@@ -453,7 +453,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             If containingType.SpecialType = SpecialType.System_Nullable_T Then
                 Return typeFromSignature Is containingType
             Else
-                Return typeFromSignature.GetNullableUnderlyingTypeOrSelf() Is containingType
+                Return typeFromSignature.GetNullableUnderlyingTypeOrSelf().GetTupleUnderlyingTypeOrSelf() = containingType.GetTupleUnderlyingTypeOrSelf()
```

Comment:
Yes, it can. This function is called at the use site to verify that a method syntactically declared as an operator has a valid signature.